lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: kubevirt
+    plugin_type: inventory
+    author:
+      - KubeVirt Team (@kubevirt)
+
+    short_description: KubeVirt inventory source
+
+    description:
+      - Fetch VirtualMachines and VirtualMachinesInstances for one or more clusters
+      - Groups by cluster name, namespace, namespace_vms, namespace_vmis  and labels
```

Comment:
What are `namespace_vms, namespace_vmis`?

Reply:
Right, `namespace_vms` should be removed .. actually even `namespace_vmis`, namespace is enough, if we don't support VMs, but only VMIs.

Reply:
This still needs addressing.

Reply:
(As in – we should drop the vm/vmis)